clients/ctl/admin-ui/src/features/config-wizard/PrivacyDeclarationForm.tsx

```diff
@@ -55,20 +49,35 @@ import {
   useGetSystemByFidesKeyQuery,
   useUpdateSystemMutation,
 } from "../system/system.slice";
-import { changeReviewStep, selectSystemFidesKey } from "./config-wizard.slice";
+import PrivacyDeclarationAccordion from "./PrivacyDeclarationAccordion";
 
-type FormValues = Partial<PrivacyDeclaration>;
+type FormValues = PrivacyDeclaration;
 
-const PrivacyDeclarationForm = ({
-  handleCancelSetup,
-}: {
-  handleCancelSetup: () => void;
-}) => {
-  const systemFidesKey = useAppSelector(selectSystemFidesKey);
-  const { data: existingSystem } = useGetSystemByFidesKeyQuery(systemFidesKey);
+const ValidationSchema = Yup.object().shape({
+  name: Yup.string().required().label("Declaration name"),
```

Comment:
I noticed I'm able to click "add another declaration" when the name is blank. This gives the "already exists" error toast, but that's not quite right.